test/CMakeLists.txt

```diff
@@ -2,5 +2,4 @@
 add_executable(testregex testregex.cpp)
 add_test(testregexcorrect testregex)
 
-
 # correctness testing of checks
```

Comment:
We should add the unit test here in the following manner:
`add_test(aliceo2namespace COMMAND ${CMAKE_BINARY_DIR}/tool/O2codecheck --checks=-*,alice* ${CMAKE_SOURCE_DIR}/test/aliceO2-namespace-naming.cpp --)` (or similar ... with your script that finally gives thumbs up or thumbs down for this test).

This will make sure the tests are run when we say `make test`.